donations/views.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import viewsets
+from .models import ThankYouNote, DonationPopup
+from .serializers import ThankYouNoteSerializer, DonationPopupSerializer
+from rest_framework.decorators import action
+from django.utils import timezone
+from django.http import JsonResponse
+
+
+class ThankYouNoteViewSet(viewsets.ModelViewSet):
+    serializer_class = ThankYouNoteSerializer
+
+    @action(methods=['post'], detail=True)
+    def post(self, request):
+        thank_you_note = request.data['thank_you_note']
+        user_info = request.data['user_info']
+        created = timezone.now()
+        try:
+            ty_note = ThankYouNote(thank_you_note=thank_you_note, user_info=user_info, created=created)
+            ty_note.save()
+        except:
```

Comment:
what is this try/except block doing? (also, good to not have blanket exceptions like this)

Reply:
I'm not sure why I did this. I left the objects.create() in and removed the rest.